L1Trigger/TrackFindingTracklet/interface/Settings.h

```diff
@@ -289,6 +289,13 @@ namespace trklet {
     void setStripLength_PS(double stripLength_PS) { stripLength_PS_ = stripLength_PS; }
     void setStripLength_2S(double stripLength_2S) { stripLength_2S_ = stripLength_2S; }
 
+    double overlapSize() const { return overlapSize_; }
+    long unsigned int numTracksPerBin() const { return numTracksPerBin_; }
```

Comment:
How can people guess that these functions have anything to do with the duplicate removal? Wherever it's not obvious what code is for, a comment is needed.

Reply:
Comments have been added.